linters/import/glide.yaml

```diff
@@ -0,0 +1,11 @@
+package: github.com/m3db/build-tools/importlint
+import:
+- package: github.com/kisielk/gotool
+  version: master
+- package: golang.org/x/tools
+  version: master
+  subpackages:
+  - go/loader
+testImport:
+- package: github.com/stretchr/testify
+  version: ">= 1.1"
```

Comment:
`>= 1.1` allows for `2.x`, probably best to stick to the same major version - i.e. `~1.1` (which is `>=1.1` and `< 2`).

Ref: https://glide.readthedocs.io/en/latest/versions/